src/story_parse.py

```diff
@@ -292,6 +327,8 @@ def __init__(self, book_urls, text_or_url, num_words, max_length,
         self.num_unique_words = len(self.decode_dict)
         self.all_sentences = self._get_all_sentences()
         self.all_stories = self._get_all_stories()
+        if embedding_path:
+            self.embeddings = self._get_embeddings(embedding_path)
```

Comment:
is self.embeddings used anywhere?

Reply:
no, im using self.embeddings as the word vectors, as opposed to self.encode_dict, which contains one hot indices